amazonlinux/README.md

```diff
@@ -73,7 +73,7 @@ Note that Docker Hub's vulnerability scanning for Amazon Linux is currently base
 ## What is Amazon Linux 2023?
 
 -	Amazon Linux 2023: https://aws.amazon.com/linux/amazon-linux-2023/
--	2023.0 Release Notes: https://docs.aws.amazon.com/linux/al2023/release-notes/relnotes.html
+-	2023.0 Release Notes: https://docs.aws.amazon.com/linux/al2023/release-notes/relnotes-2023.0.html
```

Comment:
this should be left unchanged and will be auto-generated ?

Reply:
Correct, the `README.md` must be unchanged.